Libraries/StyleSheet/processAspectRatio.js

```diff
@@ -12,10 +12,13 @@
 
 const invariant = require('invariant');
 
-function processAspectRatio(aspectRatio: number | string): ?number {
+function processAspectRatio(aspectRatio?: number | string): ?number {
   if (typeof aspectRatio === 'number') {
     return aspectRatio;
   }
+  if (typeof aspectRatio !== 'string') {
+    return;
+  }
```

Comment:
Perhaps we should an `invariant` call here when in dev mode

Reply:
I think if the user passed `number` and neither `string`, his intention would mean ignoring `aspectRatio` value like the following.

```tsx
<View style={{ aspectRatio: isDeviceLandscape ? undefined : 16 / 9 }}/>
```

The `invariant` may be used for invalid `string` or `number`.
But in this case, we can ignore it silently.

Reply:
Yeah, I'm ok with ignoring it if `aspectRatio` is `undefined`, `null` or `false` but silently ignoring `() => {}` and `, [1, 2, 3]` feels weird. IMHO we should give some feedback to the developer that they are not using this prop correctly.

There's already some logic for that in https://github.com/facebook/react-native/blob/ed5a0de7280bb08b933bcd0d6c96a516d3a328f2/Libraries/StyleSheet/processAspectRatio.js#L37-L43

Reply:
I rework about `() => {}`, `[1, 2, 3]`, `{}` cases. Thanks